coog/templates/customer-backend/deployment.yaml

```diff
@@ -65,20 +74,56 @@ spec:
                 secretKeyRef:
                   name: coog-postgres-db-secret
                   key: PASSWORD
-            - name: COOG_GATEWAY_URL
-              value: "{{ .Values.gateway.gateway_url }}{{ .Values.gateway.entrypoint }}"
-            - name: COOG_WEB_TOKEN
-            {{- if not .Values.customer_backend.coog_web_token }}
+            {{- if .Values.customer_backend.extraEnv }}
+            {{- range $key, $value := .Values.customer_backend.extraEnv }}
+            - name:  {{ $key }}
+              value: {{ $value | quote }}
+            {{- end }}
+            {{- end }}
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.customer_backend.image.repository }}:{{ .Values.customer_backend.image.tag }}"
+          imagePullPolicy: {{ .Values.customer_backend.image.pullPolicy }}
+          securityContext:
+            allowPrivilegeEscalation: false
+            capabilities:
+              drop:
+              - ALL
+            privileged: false
+            procMount: Default
+            runAsNonRoot: true
+            runAsUser: 1000
+          command: ['yarn', 'start']
+          volumeMounts:
+            - name: builddir
+              mountPath: "/workdir/build"
+          env:
+            - name:  PORT
+              value: "1337"
```

Comment:
Port of backend service.
I don't know, if, somehow, someday, someone wants to change it?

Reply:
We don't really care because on Kubernetes this is only the port for the PODs that we can easily change in service parameters and on Docker we can do the samething